src/common/components/icons/Bike.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Bus: React.FC<React.PropsWithChildren<unknown>> = () => {
```

Comment:
This name should probably be `Bike`?

Reply:
Yes, sorry about this. Same mistake was with Car also. Fixed now.